### PR TITLE
Channels v2

### PR DESCRIPTION
**Pull request by kostko**

### CODE REVIEW

File: rest_framework_reactive/models.py [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
+    STATUS_CHOICES = (
+        (STATUS_NEW, "New"),
+        (STATUS_OBSERVING, "Observing"),
+        (STATUS_STOPPED, "Stopped"),
+    )
 
+    id = models.CharField(primary_key=True, max_length=64)
+    status = models.CharField(max_length=15, choices=STATUS_CHOICES, default=STATUS_NEW)
+    request = models.BinaryField()
+    last_evaluation = models.DateTimeField(auto_now_add=True)
```
Comment (dblenkus) [inline rest_framework_reactive/models.py]: Where is this updated? Should it be `auto_now=True`?

File: rest_framework_reactive/models.py [hunk 2]
```diff
@@ -1,66 +1,71 @@
...
+    poll_interval = models.IntegerField(null=True)
+    subscribers = models.ManyToManyField('Subscriber')
 
-@dispatch.receiver(model_signals.post_save)
-def model_post_save(sender, instance, created=False, **kwargs):
-    """
-    Signal emitted after any model is saved via Django ORM.
+    def __repr__(self):
```
Comment (dblenkus) [inline rest_framework_reactive/models.py]: You can define `__str__` as:

    def __str__(self):
        return 'id={id}'.format(id=self.id)

and [default repr](https://github.com/django/django/blob/stable/1.11.x/django/db/models/base.py#L593) wraps it into `<Observer: >`

File: rest_framework_reactive/models.py (hunk 1 above)
Comment (kostko) [reply rest_framework_reactive/models.py]: Thanks, fixed.

File: docs/CHANGELOG.rst [hunk 3]
```diff
@@ -4,6 +4,15 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+==========
+Unreleased
+==========
+
+Changed
+-------
+* **BACKWARD INCOMPATIBLE:** Require Django Channels 2.1
```
[the remark this answers is not in the document]
Comment (kostko) [reply docs/CHANGELOG.rst]: Done.

File: tox.ini [hunk 4]
```diff
@@ -1,17 +1,22 @@
 [tox]
-envlist = py{27,34,35}
+envlist = py{36}
```
[the remark this answers is not in the document]
Comment (kostko) [reply tox.ini]: Done.

File: rest_framework_reactive/models.py (hunk 2 above)
Comment (kostko) [reply rest_framework_reactive/models.py]: Nice, done.

**Pull request merged**